src/ORM/Table.php

```diff
@@ -2113,6 +2117,10 @@ public function marshaller()
      */
     public function newEntity($data = null, array $options = [])
     {
+        if ($options instanceof SaveOptionsBuilder) {
```

Comment:
Since `$options` is type hinted as `array` you won't be able to use a `SaveOptionsBuilder` instance for it. Same for other methods for creating/patching entities.

The possibility of someone having overridden these methods is also quite large so removing the type hints won't be BC.